spec/jobs/get_usps_proofing_results_job_spec.rb

```diff
@@ -1148,6 +1148,56 @@
             error_type: Faraday::NilStatusError,
           )
         end
+
+        context 'when a user was flagged with fraud_pending_reason' do
+          before(:each) do
+            pending_enrollment.profile.update!(fraud_pending_reason: 'threatmetrix_review')
+            stub_request_passed_proofing_results
+          end
+
+          it 'logs something useful' do
```

Comment:
can we change this to something like `'it logs analytics events'` so it's easy to know this is where we're testing our event?

Reply:
Yes! This was a sloppy placeholder I meant to fix but had overlooked. 😄 Thanks for catching this; good names are important!